coa-helm/templates/cromwell.yaml

```diff
@@ -101,6 +102,10 @@ data:
           container: "{{ .Values.persistence.blobContainer }}"
           endpoint: "https://{{ .Values.persistence.storageAccount }}.blob.core.windows.net"
         }
+        drs {
```

Comment:
Curious if we want to enable DRS on GCP as well? I only see the `gcs` filesystem today.

https://github.com/broadinstitute/cromwhelm/blob/main/cromwell-helm/templates/cromwell.yaml#L85

Reply:
Yeah, we will want to at some point, but out of scope for this ticket.